library/std/internal.qs

```diff
@@ -163,7 +163,7 @@ namespace Microsoft.Quantum.Intrinsic {
         }
     }
 
-    @Config(Base)
+    @Config(not Adaptive)
```

Comment:
Is this change to contrast the other 'Adaptive' version? Base seems a bit easier to understand - not requiring any special capabilities.

Reply:
In this case, it's meant to be more specific. `@Config(Base)` would mean it matches only the empty set of capabilities, whereas this version of the callable should be used if the target doesn't support the `Adaptive` capability. So if the target capabilities were for some reason configured with just `QubitReset` and nothing else, this is the version that should be used rather than the one that required adaptivity.